core/primitives/piece/piece_data.hpp

```diff
@@ -28,8 +29,11 @@ namespace fc::primitives::piece {
 
     bool isOpened() const;
 
+    bool IsNullData() const;
+
    private:
     int fd_;
+    bool isNullData;
```

Comment:
Please, use suffix `_` for private members and initialize them. Also  we use snake case for data members,
```suggestion
    bool is_null_data_ = false;
```

Reply:
Fixed